services/movie_session.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+from typing import List
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(

```

Comment:
You don’t need to `return` anything in create/update/delete methods, check everywhere